sim/dex.js

```diff
@@ -1411,6 +1411,7 @@ class ModdedDex {
 			if (!format.column) format.column = column;
 			if (this.formatsCache[id]) throw new Error(`Format #${i + 1} has a duplicate ID: '${id}'`);
 			format.effectType = 'Format';
+			format.baseRuleset = format.ruleset ? format.ruleset.slice() : [];
```

Comment:
That doesn't work once <code>format.ruleset</code> is altered elsewhere, like in <code>Dex.getBanlistTable</code>.